WowPacketParser/Misc/Settings.cs

```diff
@@ -45,33 +45,17 @@ public static class Settings
         public static readonly string HotfixesDatabase = Conf.GetString("HotfixesDatabase", "hotfixes");
         public static readonly string CharacterSet = Conf.GetString("CharacterSet", "utf8");
 
-        private static UInt64 GetSQLOutputFlag()
+        private static List<string> GetSQLOutput()
```

Comment:
Why is this a `List<string>` instead of `List<SQLOutput>`? All those `ToString()` are unnecessary.